src/documents/Sonification.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Sonification
+subheader:
+layout: default
+category: Methods
+---
+
+## What is Sonification?
+
+Sonification is a general term for conveying information or data through non-speech audio &mdash; compare to the much better-known term "visualization". Many of us are familiar with simple sonification as an aspect of the user interface of computers or other digital devices, such as alert sounds when receiving new email.
+
+*Data sonification* is an analogous audio technique to *data visualization* where structured data is converted into another form in order to communicate aspects of often-complex information clearly and efficiently, to support specific analysis of aspects of the data, or to show patterns and relationships in the data.
```

Comment:
This is a hard sentence to parse. Maybe splitting into separate sentences or use a bullet list?